__tests__/app.test.js

```diff
@@ -65,6 +65,52 @@ describe("GET /api/articles", () => {
         expect(body).toBeSortedBy("created_at", { descending: true });
       });
   });
+
+  test("200: sort articles by any valid column -descending", () => {
+    return request(app)
+      .get("/api/articles?sort_by=created_at&order=desc")
+      .expect(200)
+      .then(({ body }) => {
+        //const { articles } = body;
+        expect(Array.isArray(body)).toBe(true);
+        expect(body.length).toBe(13);
+        expect(body).toBeSortedBy("created_at", { descending: true });
+      });
+  });
+
+  test("200: sort articles by any valid column - ascending", () => {
+    return request(app)
+      .get("/api/articles?sort_by=created_at&order=asc")
+      .expect(200)
+      .then(({ body }) => {
+        //const { articles } = body;
+        expect(Array.isArray(body)).toBe(true);
+        expect(body.length).toBe(13);
+        expect(body).toBeSortedBy("created_at", { ascending: true });
```

Comment:
ascending is not a value that can be used here, I would recommend visiting the docs on jest-sorted.